config_schema.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
try to remove the file and see if the make commands still work since they use the file from the oss folder